cmd/juju/caas/cluster.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package caas
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/juju/cmd"
+	"github.com/juju/utils"
+	"github.com/juju/utils/exec"
+)
+
+//go:generate mockgen -package mocks -destination mocks/runner_mock.go github.com/juju/juju/cmd/juju/caas CommandRunner
+
+type CommandRunner interface {
+	RunCommands(run exec.RunParams) (*exec.ExecResponse, error)
+}
+
+type defaultRunner struct{}
+
+func (*defaultRunner) RunCommands(run exec.RunParams) (*exec.ExecResponse, error) {
+	return exec.RunCommands(run)
+}
+
+func getEnv(key string) string {
+	result := os.Getenv(strings.ToUpper(key))
+	if result == "" {
+		result = os.Getenv(strings.ToLower(key))
+	}
+	return result
+}
+
+func kubeconfig() string {
```

Comment:
there is a `getKubeConfigPath` fun in `caas/kubernetes/clientconfig/k8s.go` uses official way to get the path in `client-go`.

Reply:
thanks, used that